src/anticodegen.c

```diff
@@ -26,10 +26,10 @@ JL_DLLEXPORT size_t jl_LLVMDisasmInstruction(void *DC, uint8_t *Bytes, uint64_t
 int32_t jl_assign_functionID(const char *fname) UNAVAILABLE
 
 void jl_init_codegen(void) { }
-void jl_fptr_to_llvm(jl_fptr_t fptr, jl_method_instance_t *lam, int specsig)
+void jl_fptr_to_llvm(void *fptr, jl_method_instance_t *lam, int specsig)
 {
     if (!specsig)
-        lam->fptr = fptr;
+        lam->specptr.fptr = fptr;
 }
```

Comment:
I think you're not supposed to implement this function anymore (just leave it with a completely empty body).